src/Microsoft.AspNetCore.Http.Abstractions/HttpMethods.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Http
+{
+    public static class HttpMethods
+    {
+        public const string Connect = "CONNECT";
+        public const string Delete = "DELETE";
+        public const string Get = "GET";
+        public const string Head = "HEAD";
+        public const string Options = "OPTIONS";
+        public const string Patch = "PATCH";
+        public const string Post = "POST";
+        public const string Put = "PUT";
+        public const string Trace = "TRACE";
```

Comment:
I'm for keeping these const. ReferenceEquals still works across assembly boundaries. Just don't mess up 😛 .